etc/schema.json

```diff
@@ -5488,7 +5497,20 @@
                 }
             },
             "additionalProperties": false
+        },
+        "websocket": {
+            "type": "object",
+            "properties": {
+                "mask": {
```

Comment:
log keyword parity would demand we log the mask :)

Reply:
The mask is both logged and detected as a boolean :-p